rpcserver.go

```diff
@@ -6669,12 +6669,23 @@ func marshalDBEdge(edgeInfo *models.ChannelEdgeInfo,
 	return edge
 }
 
+// marshalPolicyExtraOpaqueData marshals the given tlv data and filters out
+// inbound fee record.
+func marshalPolicyExtraOpaqueData(data []byte) map[uint64][]byte {
+	records := marshalExtraOpaqueData(data)
+
+	// Remove the inbound fee record as we have dedicated fields for it.
+	delete(records, uint64(lnwire.FeeRecordType))
```

Comment:
should we then maybe add to the CustomRecords proto field that it only includes records that we were not able to extract?

Reply:
Make sense! Will add!